langchain/vectorstores/faiss.py

```diff
@@ -48,12 +48,14 @@ def __init__(
         index: Any,
         docstore: Docstore,
         index_to_docstore_id: Dict[int, str],
+        normalize_score_fn: Optional[Callable[[float], float]] = None,
```

Comment:
ooc do we pass this in instead of having as method on class bc diff Faiss store can have diff score types?

Reply:
Yeah differnet score types and then the scales may differ depending on what the embeddings look like, e.g., if they're not unit norm. We could do something like `np.exp(-distance * gamma)` but then gamma is still a hyperaparameter.

Reply:
can we add a default? like the one you did in the test?

Reply:
Sure - it'll make it easier to overlook if it's not appropriate since I don't have time to make something smarter then `1-s/sqrt(2)` right now though